### ISSUE TITLE
markdown lists redesign

### ISSUE THREAD

**Issue by zavodil** [nearai/nearai#614]
Our markdown lists currently have too much space between the items. @calebjacob Could you make them more compact? This might also apply to numbered lists.

<img width="1110" alt="Screenshot 2024-12-05 at 4 31 14 PM" src="https://github.com/user-attachments/assets/f4ec55e0-55c3-4450-b1fa-81b06421fdad">

**Comment by calebjacob** [comment]
Good catch - this seems like a regression might have slipped into our styles here. I'll get this fixed.
